src/core/Form/Combobox/ComboboxItemList/ComboboxItemList.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { HtmlDiv, HtmlUlWithRef } from '../../../../reset';
+import { ComboboxItemProps } from '../ComboboxItem/ComboboxItem';
+import { baseStyles } from './ComboboxItemList.baseStyles';
+
+const baseClassName = 'fi-combobox-item-list';
+
+const comboboxItemListClassNames = {
+  wrapper: `${baseClassName}_wrapper`,
```

Comment:
ditto

Reply:
This is now also changed to be `content_wrapper`.